### ISSUE TITLE
switch sendAs live test to mock

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#1697]
> When user has only one sendAs email, do we still show it on compose? I think it could be hidden in that scenario

Currently, iOS app also indeed hides `from` field when there's only 1 sendAs email

_Originally posted by @ioanmo226 in https://github.com/FlowCrypt/flowcrypt-ios/issues/1689#issuecomment-1157771538_

We could have a mock test for that scenario too, and then switch the live test of sendAs to mock test. I'll file an issue for that now.

**Comment by sosnovsky** [comment]
Yes, `sendAs` test will be converted to mock in https://github.com/FlowCrypt/flowcrypt-ios/issues/1693
